fmdb.xcodeproj/project.pbxproj

```diff
@@ -642,6 +765,10 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = (
+					"$(PROJECT_DIR)/../../libs/SalesforceSDKCore/SalesforceSDKCore/Classes/**",
+					"$(inherited)",
+				);
```

Comment:
Shouldn't it be $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)